web/src/pages/starter-kits/[starterKitsUrl].tsx

```diff
@@ -136,24 +172,45 @@ const General = (props: InferGetStaticPropsType<typeof getStaticProps>): ReactEl
   );
 };
 
-export const getStaticProps = async (): Promise<{
-  props: {
-    noAuth: boolean;
-    roadmap: Roadmap;
+type PathParams<P> = { params: P; locale?: string };
+export type StarterKitsUrl = {
+  starterKitsUrl: string;
+};
+
+const getAllStarterKitUrls = (): PathParams<StarterKitsUrl>[] => {
+  Industries;
+  return Industries.map((industry) => {
```

Comment:
[boulder] Are we filtering only for enabled industries? We should not be generating a cemetery starter kit yet, for instance.

Reply:
Thats, a great point that I should definitely fix.

Reply:
That would be a really nice thing for `industry.ts` to handle based on the environment so we don't have to actually account for "enabled" in the code.

Reply:
I'm not exactly sure I know what you mean @jphechter

Reply:
Noting here for documentation purposes, we talked about this at dev sync today. There aren't a ton of places where we show all industries in the application, but in cases we do must account for industries that are not enabled. A nice feature might be a function call that handles the process of filtering out industries that are not enabled so that we as developers are not responsible for remembering to do this whenever we're accessing this list.